<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>Frequent Itemsets</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Find Itemsets &amp;&amp; Rules</string>
     </property>
     <widget class="QPushButton" name="btnRunApriori">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Apriori</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>32</y>
        <width>351</width>
        <height>102</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>From Invoice Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To Invoice Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinSupport">
           <property name="text">
            <string>Min. Support(%):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMaxNumItemsets">
           <property name="text">
            <string>Min. Confidence (%):</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDateEdit" name="txtFromDate">
           <property name="displayFormat">
            <string>MM/dd/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="txtToDate">
           <property name="displayFormat">
            <string>MM/dd/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="minSupportSpinBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>0.000100000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="minConfSpinBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableWidget" name="itemsetTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>471</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Itemsets</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Transactions</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Support Count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Support (%)</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Filter Itemsets</string>
     </property>
     <widget class="QPushButton" name="btnFilterItemset">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Contains:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtFilterContains"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkShow">
         <property name="text">
          <string>Show Only Support &gt;= Min. Support</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>22</y>
        <width>201</width>
        <height>74</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>From Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>To Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinSupportFilter_2">
           <property name="text">
            <string>Min. Support (%):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinSupportFilter_3">
           <property name="text">
            <string>Min. Confidence (%):</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lblFomDateFilter">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblToDateFilter">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinSupportFilter">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinConfFilter">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>601</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>Rules</string>
    </property>
    <widget class="QTableWidget" name="rulesTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>551</width>
       <height>521</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Antecedent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Consequent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Support (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Confidence (%)</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Antecedent</string>
     </property>
     <widget class="QPushButton" name="btnFilterAnt">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Contains</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtFilterContainAnt"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Consequent</string>
     </property>
     <widget class="QPushButton" name="btnFilterConseq">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Contains</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtFilterContainConsq"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionItem_List"/>
    <addaction name="actionSales_Transaction"/>
    <addaction name="actionReports"/>
    <addaction name="separator"/>
    <addaction name="actionActivity_Log"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionImport_Items"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuTool"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionImport_Items">
   <property name="text">
    <string>Import Data</string>
   </property>
  </action>
  <action name="actionImport_Transaction_Sales">
   <property name="text">
    <string>Import Transaction Sales</string>
   </property>
  </action>
  <action name="actionItem_List">
   <property name="text">
    <string>Item List</string>
   </property>
  </action>
  <action name="actionSales_Transaction">
   <property name="text">
    <string>Sales Transaction</string>
   </property>
  </action>
  <action name="actionActivity_Log">
   <property name="text">
    <string>Activity Log</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
